internal/commands/whoami/command.go

```diff
@@ -51,5 +53,24 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 	}
 
 	ui.Print(terminal.NewTextLog("Currently logged in user: %s (%s)", userDisplay, userSecret))
+
+	groups, groupsErr := clients.Atlas.Groups()
```

Comment:
[style/nit] you've been in a "named err" type of mood lately, eh?

also, b/c this is a `== nil` case, you could inline this with the `if` itself (and keep a tighter scope)